scss/_buttons.scss

```diff
@@ -72,6 +86,7 @@
     // Boosted mod: no &:focus-visible
   }
 
+  .btn-check:disabled + &,
```

Comment:
Please double-check the Boosted mods. Seems to me that some are missing in this file.